docs/api/cli.md

```diff
@@ -3,4 +3,373 @@ layout: docu
+
+```command
+D .schema
+```
+```command
+CREATE TABLE fliers(animal VARCHAR);;
+CREATE TABLE swimmers(animal VARCHAR);;
+CREATE TABLE walkers(animal VARCHAR);;
+```
+
+## Opening Database Files
+In addition to connecting to a database when opening the CLI, a new database connection can be made by using the `.open` command. If no additional parameters are supplied, a new in-memory database connection is created. This database will not be persisted when the CLI connection is closed. 
+
+```command
+D .open
+```
+
+The `.open` command optionally accepts several options, but the final parameter can be used to indicate a path to a persistent database (or where one should be created). The special string `:memory:` can also be used to open a temporary in-memory database.
+
+```command
+D .open persistent.duckdb
+```
+
+One important option accepted by `.open` is the `--readonly` flag. This disallows any editing of the database. To open in read only mode, the database must already exist. This also means that a new in-memory database can't be opened in read only mode since in-memory databases are created upon connection.
+
+```command
+D .open --readonly preexisting.duckdb
+```
+
+## Writing Results to a File
+
+By default, the DuckDB CLI sends results to the terminal's standard output. However, this can be modified using either the `.output` or `.once` commands. Pass in the desired output file location as a parameter. The `.once` command will only output the next set of results and then revert to standard out, but `.output` will redirect all subsequent output to that file location. Note that each result will overwrite the entire file at that destination. To revert back to standard output, enter `.output` with no file parameter.
+
+In this example, the output format is changed to markdown, the destination is identified as a markdown file, and then DuckDB will write the output of the SQL statement to that file. Output is then reverted to standard output using `.output` with no parameter.
+```sql
+D .mode markdown
+D .output my_results.md
+D SELECT 'taking flight' AS output_column;
+D .output
+D SELECT 'back to the terminal' as displayed_column;
+```
+
+The file my_results.md will then contain:
+```command
+| output_column |
+|---------------|
+| taking flight |
+```
+
+The terminal will then display:
+
+|   displayed_column   |
+|----------------------|
+| back to the terminal |
+
+A common output format is CSV, or comma separated values. DuckDB supports [SQL syntax to export data as CSV or Parquet](/docs/sql/statements/copy#csv-export), but the CLI-specific commands may be used to write a CSV instead if desired.
+
+```sql
+D .mode csv
+D .once my_output_file.csv
+D SELECT 1 AS col_1, 2 AS col_2
+> UNION ALL
+> SELECT 10 AS col1, 20 AS col_2;
+```
+
+The file my_output_file.csv will then contain:
+```
+col_1,col_2
+1,2
+10,20
+```
+<!-- TODO: Document .output -e and -x -->
```

Comment:
Think this is DONE now?

Reply:
> Looks great! Just a few small things. And I learned so much!

Thanks for the feedback! I'll make those changes tomorrow morning - those are all worth improving.